linchpin/api/__init__.py

```diff
@@ -1,28 +1,77 @@
 #!/usr/bin/env python
 
 import os
-
```

Comment:
The spacing here is intentional. It keeps stdlibs and linchpin libs visibly separate and makes it easier to read and recognize what libraries are being imported quickly.